docs/details/commands/pmm-agent.md

```diff
@@ -0,0 +1,149 @@
+
+`--machine-id=machine-id`
+: Node machine ID (default is auto-detected) (`PMM_AGENT_SETUP_MACHINE_ID`)
+
+`--metrics-mode=auto`
+: Metrics flow mode for agents node-exporter. Can be `push` (agent will push metrics), `pull` (server scrapes metrics from agent) or `auto` (chosen by server). (`PMM_AGENT_SETUP_METRICS_MODE`)
+
+`--node-model=NODE-MODEL`
+: Node model (`PMM_AGENT_SETUP_NODE_MODEL`)
+
+`--paths-exporters_base=PATHS-EXPORTERS_BASE`
+: Base path for exporters to use (`PMM_AGENT_PATHS_EXPORTERS_BASE`)
+
+`--paths-mongodb_exporter=PATHS-MONGODB_EXPORTER`
+: Path to `mongodb_exporter` (`PMM_AGENT_PATHS_MONGODB_EXPORTER`)
+
+`--paths-mysqld_exporter=PATHS-MYSQLD_EXPORTER`
+: Path to `mysqld_exporter` (`PMM_AGENT_PATHS_MYSQLD_EXPORTER`)
+
+`--paths-node_exporter=PATHS-NODE_EXPORTER`
+: Path to `node_exporter` (`PMM_AGENT_PATHS_NODE_EXPORTER`)
+
+`--paths-postgres_exporter=PATHS-POSTGRES_EXPORTER`
+: Path to `postgres_exporter` (`PMM_AGENT_PATHS_POSTGRES_EXPORTER`)
+
+`--paths-proxysql_exporter=PATHS-PROXYSQL_EXPORTER`
+: Path to `proxysql_exporter` (`PMM_AGENT_PATHS_PROXYSQL_EXPORTER`)
+
+`--paths-pt-summary=PATHS-PT-SUMMARY`
+: Path to pt-summary (`PMM_AGENT_PATHS_PT_SUMMARY`)
+
+`--paths-tempdir=PATHS-TEMPDIR`
+: Temporary directory for exporters (`PMM_AGENT_PATHS_TEMPDIR`)
+
+`--ports-max=PORTS-MAX`
+: Highest allowed port number for listening sockets (`PMM_AGENT_PORTS_MAX`)
+
+`--ports-min=PORTS-MIN`
+: Lowest allowed port number for listening sockets (`PMM_AGENT_PORTS_MIN`)
+
+`--region=REGION`
+: Node region (`PMM_AGENT_SETUP_REGION`)
+
+`--server-address=host:port`
+: PMM Server address and port number (`PMM_AGENT_SERVER_ADDRESS`)
+
+`--server-insecure-tls`
+: Skip PMM Server TLS certificate validation (`PMM_AGENT_SERVER_INSECURE_TLS`)
+
+`--server-password=SERVER-PASSWORD`
+: Password to connect to PMM Server (`PMM_AGENT_SERVER_PASSWORD`)
+
+`--server-username=SERVER-USERNAME`
+: Username to connect to PMM Server (`PMM_AGENT_SERVER_USERNAME`)
+
+`--skip-registration`
+: Skip registration on PMM Server (`PMM_AGENT_SETUP_SKIP_REGISTRATION`)
+
+`--trace`
+: Enable trace output (implies `--debug`) (`PMM_AGENT_TRACE`)
+
+`--version`
+: Show application version, PMM version, timestamp, git commit hash and branch.
+
+## ENVIRONMENT
+
+See OPTIONS.
+
+## LOGGING
+
+By default, pmm-agent sends messages to stderr and to the system log (syslogd or journald on Linux).
+
+To get a separate log file, edit the `pmm-agent` start-up script.
+
+- On systemd-based systems, edit `/usr/lib/systemd/system/pmm-agent.service` and change the value for `StandardError`. Example:
+
+    ```
+    StandardError=file:/var/log/pmm-agent.log
+    ```
+
+- On initd-based systems, edit `/etc/init.d/pmm-agent` and change the value for `pmm_log`. Example:
+
+    ```
+    pmm_log="/var/log/pmm-agent.log"
+    ```
+
+!!! alert alert-warning "Warning"
+    Log files are not automatically truncated or rotated. You are responsible for monitoring their size.
```

Comment:
No actions are required on initd-based systems if the log name hasn't changed. Otherwise file /etc/logrotate.d/pmm-agent-logrotate should be modified.
![Screenshot_20201217_113014](https://user-images.githubusercontent.com/25184101/102469621-a33fc880-405b-11eb-9150-9a9590beb2c9.png)

Reply:
What's the equivalent for systemd?